src/OptimisticTokenVotingPlugin.sol

```diff
@@ -257,6 +270,7 @@ contract OptimisticTokenVotingPlugin is
         }
 
         // The bridge cannot vote directly. It must use a dedicated function.
+        // Q(danielw): better to exclude taikoL1 and taikoERC20Vault as well.
```

Comment:
Ok 👍 
Question: Can `taikoBridge` be removed in general?

Reply:
To calculate L1 votes, we can ignore `taikoBridge`, but it will be used to aggregate vetos from L2.